tests/node/test_get_info.py

```diff
@@ -0,0 +1,138 @@
+"""
+Provide tests for command line interface's node get information command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_27_IN_TESTNET_ADDRESS,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_get_node_info():
+    """
+    Case: get node information.
+    Expect: flag is synced and peer count are returned.
```

Comment:
the/a flag is synced and peer count are returned.

Reply:
@Alladin9393, done.